package.json

```diff
@@ -78,7 +78,7 @@
     "fuse.js": "2.6.1",
     "get-video-id": "2.1.4",
     "globby": "6.1.0",
-    "gridicons": "2.1.3",
+    "gridicons": "3.0.2-alpha",
```

Comment:
Ideally, the prereleases should’ve been `3.0.0-alpha.1`, `3.0.0-alpha.2`. I think we can and should still have `3.0.0` be the final release.

Reply:
Looks like we can `npm unpublish gridicons@3.0.0-alpha.1` and `3.0.0-alpha.2`

Reply:
For the record, the mess I provoked publishing wrong alphas has been solved and [documented](https://github.com/Automattic/gridicons/pull/283/commits/31ccaf0c436496df7d13e3515ab54ed25bdb7988):

* unpublished gridicons 3.0.1-alpha and 3.0.2-alpha from npm (so they can be reused in the future)
* published gridicons 3.0.0-alpha.1 to npm
* changed the dependency here to 3.0.0-alpha.1

Reply:
Careful with `npm unpublish` we generally only use this when we've accidentally published something that we didn't intend to. The related version cannot be reused in the future.